homeassistant/components/wolflink/sensor.py

```diff
@@ -64,8 +64,10 @@ def name(self):
     @property
     def state(self):
         """Return the state. Wolf Client is returning only changed values so we need to store old value here."""
-        if self.wolf_object.value_id in self.coordinator.data:
-            self._state = self.coordinator.data[self.wolf_object.value_id]
+        if self.wolf_object.parameter_id in self.coordinator.data:
```

Comment:
If the parameter id is not in the fetched data should we still return the old state or return None to represent unknown state?

Reply:
No idea, but I think returning the old state is okay.

Reply:
From the perspective of cloud and their (stupid) architecture we should present the old state because they are sending only data which were changed.